action-scripts/brownie/scripts/report_gauges.py

```diff
@@ -132,12 +134,20 @@ def _parse_added_transaction(transaction: dict, **kwargs) -> Optional[dict]:
         chain, gauge, gauge_selectors
     )
 
+    addr = AddrBook("mainnet")
+    to = transaction['to']
+    to_name = addr.reversebook.get(to, "!!NOT-FOUND")
+    if to_name == "20230519-gauge-adder-v4/GaugeAdder":
```

Comment:
No point in this check. Check line 108: function will do early return if `to` field is not a GaugeAdder

Reply:
yeah but which GaugeAdder.  it's important it's the right one, and sometimes we want to know which one it is. That's why I added to.  People notice stuff this way.

Reply:
I'll update line 108 to work for any gauge addr and then leave this.  In the end part of it is about a more consistent report format.

Reply:
NO that just makes things more fragile. W ill leave the code as it is, but I think this is ok like this.  We're just normalising the tables to look similar.